### PR TITLE
Include production flag in push notification registration.

### PR DESCRIPTION
**Pull request by alloy**
This flag signals whether the build is a beta or dev build vs an app store build and is required in the beta build once https://github.com/artsy/gravity/pull/9529 is merged.

### CODE REVIEW

File: Artsy/App/ARAppDelegate+Analytics.m [hunk 1]
```diff
@@ -70,12 +70,11 @@ - (void)lookAtURLForAnalytics:(NSURL *)url
 - (void)setupAnalytics
 {
     ArtsyKeys *keys = [[ArtsyKeys alloc] init];
-    ARAppStatus *status = [[ARAppStatus alloc] init];
 
     NSString *segmentWriteKey = keys.segmentProductionWriteKey;
     NSString *environment = ADJEnvironmentProduction;
 
-    if ([status isBetaOrDev]) {
+    if (ARAppStatus.isBetaOrDev) {
```
Comment (orta) [inline Artsy/App/ARAppDelegate+Analytics.m]: class methods make it harder to test thoughhhhhh

Comment (alloy) [inline Artsy/App/ARAppDelegate+Analytics.m]: Let’s cross that bridge when we get to it? There are no tests atm.

Comment (orta) [inline Artsy/App/ARAppDelegate+Analytics.m]: :P

**Comment by orta** [comment]
:+1: Looks good to me.

**Pull request merged**